test/package.spec.ts

```diff
@@ -639,10 +640,11 @@ describe('test', async () => {
         .post('/api/groups/search')
         .send(body)
         .set('Accept', 'application/json')
-        .set('Cookie', [`sessionToken=${sessionToken}`])
         .expect(200);
       expect(res.body.offers).to.be.a('object');
 
+      sessionToken = res.headers['set-cookie'][0];
+      sessionToken = sessionToken.split('=')[1].split(';')[0];
```

Comment:
we have no session token now, plese use Cookie `.set('Cookie', [`sessionToken=${sessionToken}`])`